__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -98,11 +99,32 @@ describe('<PropertyComponent />', () => {
         }
       }
 
-      const wrapper = shallow(<PropertyComponent propertyTemplate={template} />)
+      const wrapper = shallow(<PropertyComponent propertyTemplate={template}
+                               reduxPath={["http://id.loc.gov/ontologies/bibframe/note"]}/>)
       expect(wrapper.find('Connect(InputListLOC)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLiteral)').length).toEqual(0)
     })
   })
 
+  it('error if <PropertyComponent /> is missing reduxPath props', () => {
+    const template = {
+      "propertyURI": "http://id.loc.gov/ontologies/bibframe/note",
+      "type": "resource",
+      "valueConstraint": {
+        "valueTemplateRefs": [
+          "resourceTemplate:bf2:Note"
+        ],
+        "useValuesFrom": []
+      }
+    }
+    console.error = jest.fn()
```

Comment:
Does this leak outside of this test? If so, we should restore this mock when this test finishes.

Reply:
Fixed in latest commit; stored original `console.error` and then restore it to prevent leakage as `console.error.mockRestore()` didn't seem to work.